bin/fiwareDeviceSimulatorCLI

```diff
@@ -326,12 +327,40 @@ function updateScheduledJobsInGoogleSheet() {
   }
 }
 
+/**
+ * Check if it is a .env variable
+ */
+function checkIfIsEnv(section) {
+  const sectionResult = section;
+  for (const key in section) {
+    if (typeof section[key] == 'string') {
+      const split = section[key].split('.');
+      if (split.length > 0 && split[0] === 'process') {
+        sectionResult[key] = eval(section[key]);
+      }
+    }
+  }
+  return sectionResult;
+}
+
+/**
+ * Get .env variables
+ */
+function getEnvVar(file) {
+  const result = file;
+  result.domain = checkIfIsEnv(result.domain);
+  result.contextBroker = checkIfIsEnv(file.contextBroker);
+  result.authentication = checkIfIsEnv(file.authentication);
```

Comment:
If would be great to implement a general mechanism for the whole simulation JSON file. However, if it's too complex it can be as it is now, limiting to only a subset of fields in the simulation JSON file.

Reply:
Fixed